manual_jb/content/analyses.md

```diff
@@ -3,6 +3,7 @@
 1. [Bridge damage](analyses/bridge_dmg.md)
 2. [Building cluster recovery](analyses/building_cluster_recovery)
 3. [Building damage](analyses/building_dmg)
+4. [Building Structural Damage](analyses/building_structural_dmg)
```

Comment:
Could you make the "Structural" lower case? There are some analyses where we don't follow this pattern, but should. I'll open an issue so we can address that later.